src/py_avro_schema/_schemas.py

```diff
@@ -104,6 +104,9 @@ class Option(enum.Flag):
     #: Do not populate ``doc`` schema attributes based on Python docstrings
     NO_DOC = enum.auto()
 
+    #: Use the alias specified in a Pydantic model's ``Field`` instead of the field's name
+    PYDANTIC_BY_ALIAS = enum.auto()
```

Comment:
Could we have a think about this enum member name? I appreciate you are saying that this is specific to Pydantic. However, it does not _have to be_ Pydantic only. There may be other datastructures that could be supporting aliases in future.

I would propose something like `USE_FIELD_ALIAS` or `ALIASED_NAMES` or similar?